### ISSUE TITLE
Does this work with IcedCoffeeScript?

### ISSUE THREAD

**Issue by vjpr** [ashtuchkin/errTo#2]
Seems like it does, as long as the deferral not being called doesn't cause problems.

```
class @Parser

  @getHansard: (xml, done) ->
    await xamel.parse xml, errTo done, defer json
    done null, json
```

Compiled:

```
var __iced_k, __iced_k_noop;

__iced_k = __iced_k_noop = function() {};

this.Parser = (function() {
  function Parser() {}

  Parser.getHansard = function(xml, done) {
    var json, ___iced_passed_deferral, __iced_deferrals, __iced_k,
      _this = this;
    __iced_k = __iced_k_noop;
    ___iced_passed_deferral = iced.findDeferral(arguments);
    (function(__iced_k) {
      __iced_deferrals = new iced.Deferrals(__iced_k, {
        parent: ___iced_passed_deferral,
        funcname: "Parser.getHansard"
      });
      xamel.parse(xml, errTo(done, __iced_deferrals.defer({
        assign_fn: (function() {
          return function() {
            return json = arguments[0];
          };
        })(),
        lineno: 5
      })));
      __iced_deferrals._fulfill();
    })(function() {
      return done(null, json);
    });
  };

  return Parser;

})();
```

**Comment by ashtuchkin** [comment]
`errTo` is a just plain javascript module, so I believe the question is more to the IcedCoffeeScript - is it bad to have deferrals never called.

As to the more general question - about generic error handling - see my issue maxtaco/coffee-script#35

**Comment by ashtuchkin** [comment]
I've published version 0.2 with IcedCoffeeScript support. Please see the sample at the bottom of README.md

**Comment by vjpr** [comment]
Thanks mate! Its so nice not to have to write `return next e if e` everywhere. Such as simple solution too.
